kratos/processes/process.h

```diff
@@ -92,8 +95,8 @@ class Process : public Flags
      */
     virtual Process::Pointer Create(
         Model& rModel,
-        Parameters ThisParameters
-        )
+        const Parameters ThisParameters
```

Comment:
Minor: this cannot be const otherwise one cannot get the validated parameters

Reply:
shouldn't it be stored and then updated by the validated parameters?

Reply:
the idea is that one can create the object, and then the parameters that were used in the creation contain the full set of parameters

Reply:
but then additionally it would need to be source as well.

Reply:
ValidateAndAssingDefaults modifyes the incoming object